client/question/question.js

```diff
@@ -215,6 +283,14 @@ tie.constant('PARAGRAPH_TYPE_CODE', 'code');
 tie.constant('PARAGRAPH_TYPE_SYNTAX_ERROR', 'error');
 
 /**
+ * FeedbackParagraph type that will render text to look like code.
```

Comment:
This sounds like PARAGRAPH_TYPE_CODE to me, FYI.

Reply:
They are similar, but code is used as formatting text for things like buggy output. This type indicates the display of the content can be toggled (more than just formatting text). Let me know if you want to handle this differently.

Reply:
I suppose if we merge syntax-error and output paragraphs, this will give us a feedback paragraph type whose duty is to render text with a "display more" button?

Reply:
If you're talking about syntax and output types, yes, that sounds like a possibility. I thought you were talking about syntax and code types, which I think are more different in terms behavior.

Reply:
Ah yes. I just meant the docstrings for both were the same and that implies a false equivalence (as discussed offline in chat).

Reply:
Fixed